app/src/main/java/com/ncnf/friends/ui/AddFriendFragment.java

```diff
@@ -89,14 +86,14 @@ private void searchUserWithName(String name){
                 .endAt(name + "\uf8ff");
 
 
-        FirestoreRecyclerOptions<PrivateUser> options
```

Comment:
Won't this class be changed?

Reply:
Yes it will, this PR is just to change the name of the PrivateUser class to be clearer as we continue the project !